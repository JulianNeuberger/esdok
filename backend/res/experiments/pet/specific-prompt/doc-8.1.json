{"text": "The process is triggered by the demand of a functional department to fill a post . The post is advertised , applicants apply , the applications are checked and the post is filled . The process finishes when the post was filled , precisely through the conclusion of a contract of employment .", "name": "doc-8.1", "id": "doc-8.1", "category": "", "tokens": [{"text": "The", "indexInDocument": 0, "posTag": "DT", "sentenceIndex": 0}, {"text": "process", "indexInDocument": 1, "posTag": "NN", "sentenceIndex": 0}, {"text": "is", "indexInDocument": 2, "posTag": "VBZ", "sentenceIndex": 0}, {"text": "triggered", "indexInDocument": 3, "posTag": "VBN", "sentenceIndex": 0}, {"text": "by", "indexInDocument": 4, "posTag": "IN", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 5, "posTag": "DT", "sentenceIndex": 0}, {"text": "demand", "indexInDocument": 6, "posTag": "NN", "sentenceIndex": 0}, {"text": "of", "indexInDocument": 7, "posTag": "IN", "sentenceIndex": 0}, {"text": "a", "indexInDocument": 8, "posTag": "DT", "sentenceIndex": 0}, {"text": "functional", "indexInDocument": 9, "posTag": "JJ", "sentenceIndex": 0}, {"text": "department", "indexInDocument": 10, "posTag": "NN", "sentenceIndex": 0}, {"text": "to", "indexInDocument": 11, "posTag": "TO", "sentenceIndex": 0}, {"text": "fill", "indexInDocument": 12, "posTag": "VB", "sentenceIndex": 0}, {"text": "a", "indexInDocument": 13, "posTag": "DT", "sentenceIndex": 0}, {"text": "post", "indexInDocument": 14, "posTag": "NN", "sentenceIndex": 0}, {"text": ".", "indexInDocument": 15, "posTag": ".", "sentenceIndex": 0}, {"text": "The", "indexInDocument": 16, "posTag": "DT", "sentenceIndex": 1}, {"text": "post", "indexInDocument": 17, "posTag": "NN", "sentenceIndex": 1}, {"text": "is", "indexInDocument": 18, "posTag": "VBZ", "sentenceIndex": 1}, {"text": "advertised", "indexInDocument": 19, "posTag": "VBN", "sentenceIndex": 1}, {"text": ",", "indexInDocument": 20, "posTag": ",", "sentenceIndex": 1}, {"text": "applicants", "indexInDocument": 21, "posTag": "NNS", "sentenceIndex": 1}, {"text": "apply", "indexInDocument": 22, "posTag": "VBP", "sentenceIndex": 1}, {"text": ",", "indexInDocument": 23, "posTag": ",", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 24, "posTag": "DT", "sentenceIndex": 1}, {"text": "applications", "indexInDocument": 25, "posTag": "NNS", "sentenceIndex": 1}, {"text": "are", "indexInDocument": 26, "posTag": "VBP", "sentenceIndex": 1}, {"text": "checked", "indexInDocument": 27, "posTag": "VBN", "sentenceIndex": 1}, {"text": "and", "indexInDocument": 28, "posTag": "CC", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 29, "posTag": "DT", "sentenceIndex": 1}, {"text": "post", "indexInDocument": 30, "posTag": "NN", "sentenceIndex": 1}, {"text": "is", "indexInDocument": 31, "posTag": "VBZ", "sentenceIndex": 1}, {"text": "filled", "indexInDocument": 32, "posTag": "VBN", "sentenceIndex": 1}, {"text": ".", "indexInDocument": 33, "posTag": ".", "sentenceIndex": 1}, {"text": "The", "indexInDocument": 34, "posTag": "DT", "sentenceIndex": 2}, {"text": "process", "indexInDocument": 35, "posTag": "NN", "sentenceIndex": 2}, {"text": "finishes", "indexInDocument": 36, "posTag": "NNS", "sentenceIndex": 2}, {"text": "when", "indexInDocument": 37, "posTag": "WRB", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 38, "posTag": "DT", "sentenceIndex": 2}, {"text": "post", "indexInDocument": 39, "posTag": "NN", "sentenceIndex": 2}, {"text": "was", "indexInDocument": 40, "posTag": "VBD", "sentenceIndex": 2}, {"text": "filled", "indexInDocument": 41, "posTag": "VBN", "sentenceIndex": 2}, {"text": ",", "indexInDocument": 42, "posTag": ",", "sentenceIndex": 2}, {"text": "precisely", "indexInDocument": 43, "posTag": "RB", "sentenceIndex": 2}, {"text": "through", "indexInDocument": 44, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 45, "posTag": "DT", "sentenceIndex": 2}, {"text": "conclusion", "indexInDocument": 46, "posTag": "NN", "sentenceIndex": 2}, {"text": "of", "indexInDocument": 47, "posTag": "IN", "sentenceIndex": 2}, {"text": "a", "indexInDocument": 48, "posTag": "DT", "sentenceIndex": 2}, {"text": "contract", "indexInDocument": 49, "posTag": "NN", "sentenceIndex": 2}, {"text": "of", "indexInDocument": 50, "posTag": "IN", "sentenceIndex": 2}, {"text": "employment", "indexInDocument": 51, "posTag": "NN", "sentenceIndex": 2}, {"text": ".", "indexInDocument": 52, "posTag": ".", "sentenceIndex": 2}], "mentions": [{"type": "activity", "tokenDocumentIndices": [19]}, {"type": "activity", "tokenDocumentIndices": [22]}, {"type": "activity", "tokenDocumentIndices": [27]}, {"type": "activity", "tokenDocumentIndices": [32]}, {"type": "actor", "tokenDocumentIndices": [5, 6, 7, 8, 9, 10]}, {"type": "actor", "tokenDocumentIndices": [21]}, {"type": "activity data", "tokenDocumentIndices": [16, 17]}, {"type": "activity data", "tokenDocumentIndices": [29, 30]}, {"type": "activity data", "tokenDocumentIndices": [24, 25]}, {"type": "further specification", "tokenDocumentIndices": [44, 45, 46, 47, 48, 49, 50, 51]}], "entities": [{"mentionIndices": []}, {"mentionIndices": [6, 7]}, {"mentionIndices": [0]}, {"mentionIndices": [1]}, {"mentionIndices": [2]}, {"mentionIndices": [3]}, {"mentionIndices": [4]}, {"mentionIndices": [5]}, {"mentionIndices": [8]}, {"mentionIndices": [9]}], "relations": [{"headMentionIndex": 0, "tailMentionIndex": 1, "type": "flow"}, {"headMentionIndex": 1, "tailMentionIndex": 2, "type": "flow"}, {"headMentionIndex": 2, "tailMentionIndex": 3, "type": "flow"}, {"headMentionIndex": 0, "tailMentionIndex": 4, "type": "actor recipient"}, {"headMentionIndex": 0, "tailMentionIndex": 6, "type": "uses"}, {"headMentionIndex": 1, "tailMentionIndex": 5, "type": "actor performer"}, {"headMentionIndex": 1, "tailMentionIndex": 8, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 8, "type": "uses"}, {"headMentionIndex": 3, "tailMentionIndex": 7, "type": "uses"}, {"headMentionIndex": 3, "tailMentionIndex": 9, "type": "further specification"}]}