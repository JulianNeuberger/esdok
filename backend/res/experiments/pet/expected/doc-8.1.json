{"nodes": [{"id": "0", "name": "demand", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "1", "name": "a functional department", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "2", "name": "to fill a post", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "3", "name": "The post", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "4", "name": "advertised", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "5", "name": "applicants", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "6", "name": "apply", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "7", "name": "the applications", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "8", "name": "checked", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, {"id": "9", "name": "filled", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}], "edges": [{"id": "0", "source": {"id": "0", "name": "demand", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "to fill a post", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "1", "source": {"id": "0", "name": "demand", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "1", "name": "a functional department", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "2", "source": {"id": "0", "name": "demand", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "advertised", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "3", "source": {"id": "4", "name": "advertised", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "The post", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "4", "source": {"id": "4", "name": "advertised", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "apply", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "5", "source": {"id": "6", "name": "apply", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "applicants", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "6", "source": {"id": "6", "name": "apply", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "8", "name": "checked", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "7", "source": {"id": "8", "name": "checked", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "7", "name": "the applications", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "8", "source": {"id": "8", "name": "checked", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "9", "name": "filled", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "9", "source": {"id": "9", "name": "filled", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "The post", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-8.1", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}]}